docs/feature_space_cadet.md

```diff
@@ -0,0 +1,59 @@
+# Space Cadet Shift: The Future, Built In
```

Comment:
```suggestion
# Space Cadet: The Future, Built In
```

If this page is for the Space Cadet feature in general, the title probably shouldn't refer to the Shift part.

Reply:
good catch